chemicaltoolbox/openbabel/ob_grep.xml

```diff
@@ -3,8 +3,9 @@
     <!--parallelism method="multi" split_inputs="infile" split_mode="to_size" split_size="10000" shared_inputs="" merge_outputs="outfile"></parallelism-->
     <macros>
         <import>macros.xml</import>
-        <token name="@GALAXY_VERSION@">0</token>
+        <token name="@GALAXY_VERSION@">1</token>
     </macros>
+    <options sanitize="False"/>
```

Comment:
should this be moved the to actual parameter that needs it? and whitespace does not need to be allowed I guess?